incubator/nodejs-express/templates/simple/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodejs-express-simple",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
a one liner reason for this bump pls?

Reply:
@gireeshpunathil This is an auto generated PR. Don't know why it updated the version.

Reply:
Yeah, increasing the major version seems odd. Perhaps we should close this automated PR and do something equivalent ourselves (if appropriate).